README.md

```diff
@@ -16,6 +16,13 @@ Usage
 
     curl https://raw.githubusercontent.com/mikepurvis/ros-install-osx/master/install | bash
 
+or
+
+```shell
+curl -O https://raw.githubusercontent.com/mikepurvis/ros-install-osx/master/install
+bash install
```

Comment:
Sounds good. 

Honestly, I can't even remember why your original command didn't work for me. Piping to `bash` seems to work fine at the moment. 

But I will say that downloading the script/repo is nice for making minor changes to it as you go (like the hints I added to Troubleshooting).